src/js/views/patients/sidebar/action/action-sidebar.scss

```diff
@@ -22,6 +22,31 @@
   }
 }
 
+.action-sidebar__activity {
+  .spinner {
+    margin-top: 100px;
+  }
+}
+
+.action-sidebar__attachment-filename {
```

Comment:
Let's keep these in mind if we ever see any of these styles reused.  They seem generic-y